src/uu/stty/src/stty.rs

```diff
@@ -889,6 +907,124 @@ fn string_to_control_char(s: &str) -> Result<u8, ControlCharMappingError> {
+fn combo_to_flags(combo: &str) -> Vec<ArgOptions> {
+    let mut flags = Vec::new();
+    let mut ccs = Vec::new();
+    match combo {
+        "lcase" | "LCASE" => {
+            flags = vec!["xcase", "iuclc", "olcuc"];
+        }
+        "-lcase" | "-LCASE" => {
+            flags = vec!["-xcase", "-iuclc", "-olcuc"];
+        }
+        "cbreak" => {
+            flags = vec!["-icanon"];
+        }
+        "-cbreak" => {
+            flags = vec!["icanon"];
+        }
+        "cooked" | "-raw" => {
+            flags = vec![
+                "brkint", "ignpar", "istrip", "icrnl", "ixon", "opost", "isig", "icanon",
+            ];
+            ccs = vec![(S::VEOF, "^D"), (S::VEOL, "")];
+        }
+        "crt" => {
+            flags = vec!["echoe", "echoctl", "echoke"];
+        }
+        "dec" => {
+            flags = vec!["echoe", "echoctl", "echoke", "-ixany"];
+            ccs = vec![(S::VINTR, "^C"), (S::VERASE, "^?"), (S::VKILL, "^U")];
+        }
+        "decctlq" => {
+            flags = vec!["ixany"];
+        }
+        "-decctlq" => {
+            flags = vec!["-ixany"];
+        }
+        "ek" => {
+            ccs = vec![(S::VERASE, "^?"), (S::VKILL, "^U")];
+        }
+        "evenp" | "parity" => {
+            flags = vec!["parenb", "-parodd", "cs7"];
+        }
+        "-evenp" | "-parity" => {
+            flags = vec!["-parenb", "cs8"];
+        }
+        "litout" => {
+            flags = vec!["-parenb", "-istrip", "-opost", "-cs8"];
+        }
+        "-litout" => {
+            flags = vec!["parenb", "istrip", "opost", "cs7"];
+        }
+        "nl" => {
+            flags = vec!["-icrnl", "-onlcr"];
+        }
+        "-nl" => {
+            flags = vec!["icrnl", "-inlcr", "-igncr", "onlcr", "-ocrnl", "-onlret"];
+        }
+        "oddp" => {
+            flags = vec!["parenb", "parodd", "cs7"];
+        }
+        "-oddp" => {
+            flags = vec!["-parenb", "cs8"];
+        }
+        "pass8" => {
+            flags = vec!["-parenb", "-istrip", "cs8"];
+        }
+        "-pass8" => {
+            flags = vec!["parenb", "istrip", "cs7"];
+        }
+        "raw" | "-cooked" => {
+            flags = vec![
+                "-ignbrk", "-brkint", "-ignpar", "-parmrk", "-inpck", "-istrip", "-inlcr",
+                "-igncr", "-icrnl", "-ixon", "-ixoff", "-icanon", "-opost", "-isig", "-iuclc",
+                "-xcase", "-ixany", "-imaxbel",
+            ];
+            ccs = vec![(S::VMIN, "1"), (S::VTIME, "0")];
+        }
+        "sane" => {
+            flags = vec![
+                "cread", "-ignbrk", "brkint", "-inlcr", "-igncr", "icrnl", "icanon", "iexten",
+                "echo", "echoe", "echok", "-echonl", "-noflsh", "-ixoff", "-iutf8", "-iuclc",
+                "-xcase", "-ixany", "imaxbel", "-olcuc", "-ocrnl", "opost", "-ofill", "onlcr",
+                "-onocr", "-onlret", "n10", "cr0", "tab0", "bs0", "vt0", "ff0", "isig", "-tostop",
```

Comment:
I think it's `nl0`, not `n10`:
```suggestion
                "-onocr", "-onlret", "nl0", "cr0", "tab0", "bs0", "vt0", "ff0", "isig", "-tostop",
```